libraries/AP_IQ_Motor/AP_IQ_Motor.cpp

```diff
@@ -80,8 +76,8 @@ void AP_IQ_Motor::init(void)
 
 uint8_t AP_IQ_Motor::add_client(ClientAbstract *new_client)
 {
-    if (_client_size < _client_limit) // TODO check if this client type already exists and don't add if it does. Instead return the pointer?
-    {
+    //Make sure that there is enough space to add a new client
+    if (_client_size < _client_limit) { // TODO check if this client type already exists and don't add if it does. Instead return the pointer?
```

Comment:
Is doing this TODO something that is going to happen in one of your upcoming PRs?

Reply:
I was not planning on working on any of the TODOs in this sprint, but want to keep them around for future development. I think trying to further development would be feature creep beyond the defined task of "clean-up and test existing IFCI ardupilot code."